### ISSUE TITLE
PGFPlotsX support?

### ISSUE THREAD

**Issue by AshtonSBradley** [sisl/juliaplots.sty#4]
Would it be possible to support PGFPlotsX? Seems close since the API is similar ...

**Comment by mykelk** [comment]
Yeah! That would be awesome. It would involve adding about 8 lines of code. Anyone should feel free to take a stab at it and file a PR.
